package.json

```diff
@@ -48,10 +48,11 @@
     "pretest": "yarn lint",
     "test": "node ./test/runTest.js",
     "test:update": "SNAPSHOT_UPDATE=true node ./test/runTest.js",
-    "dev": "NODE_ENV=dev node ./scripts/build.js",
-    "watch": "NODE_ENV=dev,watch node ./scripts/build.js",
-    "build": "NODE_ENV=prod node ./scripts/build.js",
-    "package": "yarn build && vsce package --yarn",
+    "dev": "esbuild ./src/extension.js --bundle --platform=node --target=node16 --external:vscode --external:@typescript-eslint/parser --external:vue-eslint-parser --outfile=./dist/extension.js",
+    "dev:watch": "yarn dev --watch --sourcemap",
+    "build": "yarn dev --minify",
+    "build:worker": "esbuild ./src/worker.js --bundle --platform=node --target=node16 --outfile=./dist/worker.js",
```

Comment:
You don't need to compile the worker.

Reply:
@JounQin I have to bundle synckit that is why I'm compiling it.

Reply:
You don't need to.

Reply:
@JounQin Why don't I need to bundle it?

Reply:
Can't we mark it as dependency?

Reply:
@JounQin I could but I'm trying to keep the amount of dependencies the user has to install to a minimum.

Reply:
OK, that may be valid thought, but it complexes the whole workflow, I don't think that worth too much.

Reply:
Bundling is required by vscode even for `dependencies`, see https://github.com/microsoft/vscode/issues/200433 for tracking.